src/modules/functions.js

```diff
@@ -323,6 +323,328 @@ export const displayUserNameOverlay = () => {
   playerHeaderTarget.insertAdjacentElement("beforebegin", userOverlayContainer);
 };
 
+export const toggleTokenConversion = (toggle) => {
+  const tokenToUsdRate = 0.0828; // Conversion rate from tokens to USD
```

Comment:
what is this based on? the usd -> token conversion is different based on how many tokens you buy at once.  i haven't tested this yet, but if i decide to include it, i'd like to include a disclaimer about what rate it's based on and that it won't necessarily reflect the amount paid for those tokens

Reply:
Its an average of all the different rates the site offers, I added an explanation about it and an option to change it